### PR TITLE
added ability to search different cities within the api

### PR DESCRIPTION
**Pull request by jackmallahan**
SEMI COLONS!

### CODE REVIEW

**Comment by christielynam** [comment]
ugh

File: lib/Components/Search/Search.js [hunk 1]
```diff
@@ -4,15 +4,38 @@ import './Search.css'
...
   }
 
+  handleChange(e) {
+    this.setState({
+      input: e.target.value,
+    })
+  }
+
+  getLocation(userInput) {
+    let splitInput = userInput.split(', ');
```
Comment (mlimberg) [inline lib/Components/Search/Search.js:19]: @christielynam I don't think we need this anymore, thoughts?? The api is super loose and let's us do whatever the F we want.

**Review by mlimberg — COMMENTED** [review]
I agree, let's ditch this thang!

**Comment by mlimberg** [comment]
👍 merge it!

**Pull request merged**